src/style.css

```diff
@@ -1 +1,64 @@
-/* Styles go here */
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  --color-link: #00acdc;
+}
+
+.nav__list {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+  display: flex;
+}
+
+a {
```

Comment:
change to class and add hover to class

Reply:
I changed it, but it still the same problem